docs/gates.md

```diff
@@ -20,87 +20,109 @@ nav_order: 3
+It is a device which performs logic operations on one or more logical inputs, and produces a single logical output.
+Logic gates can be categorized into 3 groups:
+1. Basic Gates: NOT, AND, OR
+2. Universal Gates: NAND, NOR
+3. Arithmetic Gates: X-OR, X-NOR
+
+<h2>Truth Table</h2>
+
+The Table which contains all logical possibilities is known as truth table.
 
 ## NOT gate
 
-The NOT gate is also known as an inverter because the output is the exact opposite of the input. It has one input and one output. The two possibilities are written out in the table below. Tables listing all logical possibilities like this are known as truth tables.
+The NOT gate is also known as an inverter because it inverts the logic sense of a binary variable. It has one input and one output. The Truth table for NOT gate is given below
 
-| Input1       | Input 2      |
+| Input        | Output     |
 |:------------:|:------------:|
 | 0            | 1            | 
 | 1            | 0            | 
 
-
-<iframe width="100%" height="220px" src="https://circuitverse.org/simulator/embed/738" id="projectPreview" scrolling="no" webkitAllowFullScreen mozAllowFullScreen allowFullScreen> </iframe>
+<iframe width="100%" height="220px" src="https://circuitverse.org/simulator/embed/46600" id="projectPreview" scrolling="no" webkitAllowFullScreen mozAllowFullScreen allowFullScreen> </iframe>
 
 ## AND gate
 
-The AND gate has two inputs and one output. The output is 1 if both inputs are 1, and for all other cases the output is 0.
+Basically, the AND gate's operation is similar to that of multiplication. It has two inputs and one output. The output is high (1) if both inputs are 1, and for all other cases, the output is low (0).
+The Truth table for AND gate which consists of two inputs is given below
 
-| Input1       | Input 2      | output |
+| Input 1       | Input 2      | Output |
 |:------------:|:------------:|:------:|
 | 0            | 0            | 0      |
 | 0            | 1            | 0      |
 | 1            | 0            | 0      |
 | 1            | 1            | 1      |
 
-<iframe width="100%" height="220px" src="https://circuitverse.org/simulator/embed/734" id="projectPreview" scrolling="no" webkitAllowFullScreen mozAllowFullScreen allowFullScreen> </iframe>
-
-
+<iframe width="100%" height="220px" src="https://circuitverse.org/simulator/embed/46601" id="projectPreview" scrolling="no" webkitAllowFullScreen mozAllowFullScreen allowFullScreen> </iframe>
 
 ## OR gate
 
-The OR gate has two inputs and one output. If at least one of the inputs is 1, then the output will be 1. If neither input is 1, the output will be 0.
+Basically, the OR gate's operation is similar to that of addition. It has two inputs and one output. If at least one of the inputs is 1, then the output will be high (1). If neither of the inputs is 1, the output will be low (0).
```

Comment:
Remove Basically

Reply:
The statement of being similar to addition appears a little confusing as well [In case for 1 and 1]. What say?

Reply:
yep, I will change it.